src/components/frontend-engine/frontend-engine.tsx

```diff
@@ -227,6 +229,11 @@ const FrontendEngineInner = forwardRef<IFrontendEngineRef, IFrontendEngineProps>
 			const subscription = watch(() => {
 				onValueChange(getFormValues(undefined, stripUnknown), checkIsFormValid());
 			});
+
+			if (previousFormValidationConfig && previousFormValidationConfig !== formValidationConfig) {
```

Comment:
can leave a comment to explain why we need the manual call

Reply:
can we add another condition to fire onValueChange only if the form validity has changed? then we can also bring back the schema change unit test